services/dialog/v1.js

```diff
@@ -86,57 +86,57 @@ module.exports = function (RED) {
+      else if (config.mode === 'deleteall') {
         performDeleteAll(node,dialog,msg,config);
       }
-       else if (config.mode === "list") {
+       else if (config.mode === 'list') {
         performList(node,dialog,msg);
-      } else if (config.mode === "startconverse" || config.mode === "converse" || config.mode === "getprofile") {
+      } else if (config.mode === 'startconverse' || config.mode === 'converse' || config.mode === 'getprofile') {
           var dialogid = config.dialog;
           var clientid = config.clientid;
           var converseid = config.converse;
 
-          if (!dialogid || "" == dialogid) {
-            if (msg.dialog_params && "dialog_id" in msg.dialog_params) {
-              dialogid = msg.dialog_params["dialog_id"];
+          if (!dialogid || '' == dialogid) {
+            if (msg.dialog_params && 'dialog_id' in msg.dialog_params) {
+              dialogid = msg.dialog_params['dialog_id'];
             }	
           }
 		  
-          if (!dialogid || "" == dialogid) {
-            message = "Missing Dialog ID";
-            node.status({fill:"red", shape:"dot", text:message});	
+          if (!dialogid || '' == dialogid) {
+            message = 'Missing Dialog ID';
+            node.status({fill:'red', shape:'dot', text:message});	
             node.error(message, msg);	
           }			
 			
-          if (config.mode === "converse"  || config.mode === "getprofile") {
-            if (!clientid || "" == clientid) {
-              if (msg.dialog_params && "client_id" in msg.dialog_params) {
-                clientid = msg.dialog_params["client_id"];
+          if (config.mode === 'converse'  || config.mode === 'getprofile') {
+            if (!clientid || '' == clientid) {
+              if (msg.dialog_params && 'client_id' in msg.dialog_params) {
+                clientid = msg.dialog_params['client_id'];
               }	
             }				  
-            if (!converseid || "" === converseid) {
-              if (msg.dialog_params && "converse_id" in msg.dialog_params) {
-                converseid = msg.dialog_params["converse_id"];
+            if (!converseid || '' === converseid) {
+              if (msg.dialog_params && 'converse_id' in msg.dialog_params) {
+                converseid = msg.dialog_params['converse_id'];
               }	
             }				  
-		    if (!clientid || "" === clientid) {
-              message = "Missing Client ID";
-              node.status({fill:"red", shape:"dot", text:message});	
+		    if (!clientid || '' === clientid) {
+              message = 'Missing Client ID';
+              node.status({fill:'red', shape:'dot', text:message});	
               node.error(message, msg);	
             }
-            if (!converseid || "" === converseid) {
-              var message = "Missing Conversation ID";
-              node.status({fill:"red", shape:"dot", text:message});	
+            if (!converseid || '' === converseid) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["message" is already defined](https://www.codacy.com/app/rezgui-y/node-red-node-watson/file/2462016486/issues/source?bid=3225115&fileBranchId=3255594#l138)